packages/core/src/base-chart.ts

```diff
@@ -951,4 +951,12 @@ export class BaseChart {
 	getBBox(selector: any) {
 		return this.innerWrap.select(selector).node().getBBox();
 	}
+
+	// ================================================================================
+	// Makes chart data components more screen reader friendly
+	// ================================================================================
+	makeAccessible (innerWrap: any) {
+		// Make chart data components tabbable
+		innerWrap.attr("tabindex", 0);
```

Comment:
This function should be able to handle all sorts of chart components. Keep in mind the d3 `.call` function passes in the selection, therefore you can use that info in this function.